src/bin/bat/clap_app.rs

```diff
@@ -384,7 +384,7 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                 .validator(|val| {
                     let mut invalid_vals = val.split(',').filter(|style| {
                         !&[
-                            "auto", "full", "plain", "header", "grid", "rule", "numbers", "snip",
+                            "auto", "full", "plain", "header", "header-full", "header-filename", "header-filesize", "header-permissions", "header-lastmodified", "grid", "rule", "numbers", "snip",
```

Comment:
Sorry to nit-pick about formatting (I try to avoid nit-picking), but can you split this up into many lines please? `cargo fmt` seems to be OK with having it like this instead, which is more manageable:

```
                            "auto",
                            "full",
                            "plain",
                            "header",
                            "header-full",
                            "header-filename",
                            "header-filesize",
                            "header-permissions",
                            "header-lastmodified",
                            "grid",
                            "rule",
                            "numbers",
                            "snip",
                            #[cfg(feature = "git")]
                            "changes",
```